pom.xml

```diff
@@ -23,7 +23,7 @@
     <parquet.version>1.8.1</parquet.version>
     <!-- Edit the following line to configure the Hadoop (HDFS) version. -->
     <hadoop.version>2.2.0</hadoop.version>
-    <scoverage.version>0.99.2</scoverage.version>
+    <scoverage.version>1.1.1</scoverage.version>
```

Comment:
I see as of ```0.99.2``` support for scala 2.10 was dropped.
https://github.com/scoverage/scalac-scoverage-plugin/blob/master/2.10.md

Updating to ```1.1.1``` as this pull request does results in a maven transitive dependency error.

Versions ```0.99.2``` and ```0.99.10``` fail with
```adam-core/target/scoverage.coverage.xml (No such file or directory)```

Downgrading to ```0.98.2``` fails with
```adam-core/target/scala-2.10.4/classes/scoverage.coverage (No such file or directory)```